server/app.js

```diff
@@ -45,6 +46,7 @@ app.use((req, res, next) => {
 app.use("/auth", authRouter);
 app.use("/users", userRouter);
 app.use("/profile", profileRouter);
+app.use("/upload", uploadRouter);
```

Comment:
```suggestion
app.use("/uploads", uploadRouter);
```
Let's rename this route as per REST guidelines: https://blog.dreamfactory.com/best-practices-for-naming-rest-api-endpoints/